File: javascript/1-evenOrOdd.js
// function evenOrOdd(number) {
//     if (number % 2 === 0) {
//         number = 'Even';
//     }
//     else {
//         number = 'Odd';
//     }
//     return number;
// }

// function evenOrOdd(number) {
//     return number % 2 === 0 ? 'even' : 'odd';
// }

// ARROW FUNCTION
const evenOrOdd = (number) =>  number % 2 === 0 ? 'even' : 'odd';

console.log(evenOrOdd(2)); // even
console.log(evenOrOdd(3)); // odd

File: python/1-even_or_odd.py
# def even_or_odd(number):
#     if number % 2 == 0 :
#         number = 'Even'
#     else :
#         number = 'Odd'
#     return number

# ALTER
# def even_or_odd(number):
#     return 'Even' if number % 2 == 0 else 'Odd'

# LAMBDA FUNCTION
even_or_odd = lambda number : 'Even' if number % 2 == 0 else 'Odd'

print(even_or_odd(4))